modules/Kconfig.nuvoton

```diff
@@ -18,3 +18,6 @@ config HAS_NUMICRO_UART
 	  module driver
 
 endmenu
+
+config HAS_NPCX_HAL
+	bool "Nuvoton NPCX EC HAL drivers support"
```

Comment:
Can you please add help with what this means and what is supported?

Reply:
Done.

Reply:
If I move the register definitions file to soc/arm/nuvoton_npcx/common/reg, is it OK to you?

Reply:
I think that is a good place. Thank you!